moped-editor/src/views/projects/projectView/ProjectComponents/utils/useComponentLinkParams.js

```diff
@@ -63,7 +63,7 @@ export const useComponentLinkParams = ({
     }
 
     // Set clicked component from search parameter once projectComponents data loads
-    if (projectComponents.length > 0) {
+    if (projectComponents.length > 0 || allRelatedComponents.length > 0) {
```

Comment:
if a project only had related components mapped, it would skip this part of the code and not set the clicked component nor show an error.

Reply:
When I was testing, i noticed that, if the last component in one project is moved to another project, the alert doesn't show because of the `projectComponents > 0`  check that we are using to make sure the data has loaded here before checking if the param component id is in the returned data or not.

This is an edge case so, while it would be cool to find out, I don't know if it is worth figuring out. what do you think?